frontend/inbox/src/pages/Inbox/Messenger/ConversationMetadata/index.tsx

```diff
@@ -142,6 +144,7 @@ const ConversationMetadata = (props: ConnectedProps<typeof connector>) => {
 
   const saveEditDisplayName = () => {
     updateConversationContactInfo(conversation.id, displayName);
+    updateContactDetails(conversation.id, {id: contacts[conversation.id].id, displayName: displayName});
```

Comment:
What if the contact is not in the store? 🤔

Reply:
the `ContactDetails ` component fetches the contact for each conversation and puts it in the store when it is rendered, so the contact will be in the store by the time the user updates the display name.